protocol_controller/config.py

```diff
@@ -6,11 +6,18 @@ class Config:
     TESTING = False
     LOG_LEVEL = os.environ.get('LOG_LEVEL', 'DEBUG')
     API_PREFIX = os.environ.get('API_PREFIX', '/sas/v1')
+    RC_RESPONSE_WAIT_TIMEOUT = int(os.environ.get('RC_RESPONSE_WAIT_TIMEOUT', 5))  # TODO Set 5 for testing only
```

Comment:
Change the default timeout to 60.
5 is too short as request processing happens every 10sec currently.

Reply:
done